xbmc/addons/include/xbmc_epg_types.h

```diff
@@ -75,6 +75,12 @@ extern "C" {
     time_t        startTime;           /*!< @brief (required) start time in UTC */
     time_t        endTime;             /*!< @brief (required) end time in UTC */
     const char *  strPlotOutline;      /*!< @brief (optional) plot outline */
+    const char *  strShowID;           /*!< @brief (optional) showid */
```

Comment:
Changing any of these files means you'll have to bump the API version number too.